src/View/Widget/SelectBoxWidget.php

```diff
@@ -228,14 +228,14 @@ protected function _renderOptgroup(
      *
      * Will recursively call itself when option groups are in use.
      *
-     * @param array|\ArrayAccess  $options The options to render.
-     * @param array|\ArrayAccess|null $disabled The options to disable.
+     * @param array $options The options to render.
+     * @param array|null $disabled The options to disable.
      * @param array|string|null $selected The options to select.
      * @param array $templateVars Additional template variables.
      * @param bool $escape Toggle HTML escaping.
      * @return array Option elements.
      */
-    protected function _renderOptions($options, $disabled, $selected, $templateVars, $escape): array
+    protected function _renderOptions($options, ?array $disabled, $selected, $templateVars, $escape): array
```

Comment:
Could disabled be an arrayaccess if an orm result found it's way down here?

Reply:
Nope, an entity never makes it's way as argument to this function.